src/js/mutations.test.ts

```diff
@@ -41,7 +42,7 @@ const tagSet = [
     Value: 'Veeam 12',
   },
 ];
-const veeamPolicyName = `${VEEAM_IMMUTABLE_POLICY_NAME}-${bucketName}`;
+export const veeamPolicyName = `${VEEAM_IMMUTABLE_POLICY_NAME}-${bucketName}`;
```

Comment:
nit: Why is this export necessary ?

Reply:
Fixed f7a6e20110bf83c0ee8d5b8839a6f7a1919bab36